API/pubsub/README.md

```diff
@@ -0,0 +1,63 @@
+pubsub API
+==========
+
+#### `pubsub.subscribe`
+
+> Subscribe to an IPFS Topic
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.pubsub.subscribe(topic, options, callback)
+
+- `topic` - type: String
+- `options` - type: Object, optional, might contain the following properties:
+  - `discover`: type: Boolean - Will use the DHT to find 
+
+`callback` must follow `function (err, subscription) {}` where Subscription is a Node.js Stream in Object mode, emiting a `data` event for each new message on the subscribed topic.`err` is an error if the operation was not successful.
```

Comment:
@dignifiedquire - Awesome. Let me know if there's any areas you need help / support in this!